src/components/Reports/SeedbankSection.tsx

```diff
@@ -0,0 +1,189 @@
+          />
+        ) : (
+          <OverviewItemCard
+            isEditable={false}
+            title={strings.REPORT_SEEDBANK_OPERATION_START_DATE}
+            contents={seedbank.operationStartedDate ?? ''}
+            className={classes.infoCardStyle}
+          />
+        )}
+      </Grid>
+      <Grid item xs={12}>
+        <OverviewItemCard
+          isEditable={false}
+          title={strings.TOTAL_SEEDS_STORED}
+          contents={seedbank.totalSeedsStored.toString() ?? '0'}
+          className={classes.infoCardStyle}
+        />
+      </Grid>
+      <Grid item xs={smallItemGridWidth()}>
+        {editable ? (
+          <Textfield
+            label={strings.WORKERS_PAID_ENGAGED}
+            id={`${seedbank.id}-workers-paid-engaged`}
+            type='number'
+            value={workersPaidEngaged}
+            readonly={!editable}
+            onChange={(value) => setWorkersPaidEngaged(value as number)}
+          />
+        ) : (
+          <OverviewItemCard
+            isEditable={false}
+            title={strings.WORKERS_PAID_ENGAGED}
+            contents={seedbank.workers.paidWorkers?.toString() ?? '0'}
+            className={classes.infoCardStyle}
+          />
+        )}
+      </Grid>
+      <Grid item xs={smallItemGridWidth()}>
+        {editable ? (
+          <Textfield
+            label={strings.WORKERS_PAID_FEMALE}
+            id={`${seedbank.id}-workers-paid-female`}
+            type='number'
+            value={workersPaidFemale}
+            readonly={!editable}
+            onChange={(value) => setWorkersPaidFemale(value as number)}
+          />
+        ) : (
+          <OverviewItemCard
+            isEditable={false}
+            title={strings.WORKERS_PAID_FEMALE}
+            contents={seedbank.workers.femalePaidWorkers?.toString() ?? '0'}
+            className={classes.infoCardStyle}
+          />
+        )}
+      </Grid>
+      <Grid item xs={smallItemGridWidth()}>
+        {editable ? (
+          <Textfield
+            label={strings.WORKERS_VOLUNTEERS}
+            id={`${seedbank.id}-workers-volunteer`}
+            type='number'
+            value={workersVolunteer}
+            readonly={!editable}
+            onChange={(value) => setWorkersVolunteer(value as number)}
+          />
+        ) : (
+          <OverviewItemCard
+            isEditable={false}
+            title={strings.WORKERS_VOLUNTEERS}
+            contents={seedbank.workers.volunteers?.toString() ?? '0'}
+            className={classes.infoCardStyle}
+          />
+        )}
+      </Grid>
+      <Grid item xs={mediumItemGridWidth()}>
+        <Textfield
+          label={strings.SEED_BANK_NOTES}
+          id={`${seedbank.id}-notes`}
+          type='textarea'
+          disabled={!editable}
+          value={seedbankNotes}
+          onChange={(value) => setSeedbankNotes(value as string)}
+        />
+        <Typography color={theme.palette.TwClrTxtSecondary} fontSize='14px' fontWeight={400}>
```

Comment:
margin top 8px?

Reply:
Hm, maybe 4px according to the figma

Reply:
anything works, seemed a bit squished in the screenshot